UI/window-basic-main.cpp

```diff
@@ -3921,6 +3921,7 @@ void OBSBasic::ClearSceneData()
 		return true;
 	};
 
+	obs_enum_scenes(cb, nullptr);
```

Comment:
This should perhaps be split into a separate commit, as it is unrelated to saving removed sources.